specs/spec_reader.go

```diff
@@ -140,10 +145,16 @@ func (r *SpecReader) validate() error {
 	return nil
 }
 
+func (r *SpecReader) GetDestinationByName(name string) *Destination {
+	return r.destinations[name]
```

Comment:
`nil` values are handled by the CLI, see https://github.com/cloudquery/cloudquery/blob/d75df29d468a84724229ecef6ce558d31e665707/cli/cmd/sync.go#L71